Lib/test/test_os.py

```diff
@@ -3066,9 +3066,11 @@ class PidTests(unittest.TestCase):
     def test_getppid(self):
         p = subprocess.Popen([sys.executable, '-c',
                               'import os; print(os.getppid())'],
-                             stdout=subprocess.PIPE)
-        stdout, _ = p.communicate()
+                             stdout=subprocess.PIPE,
+                             stderr=subprocess.PIPE)
+        stdout, error = p.communicate()
         # We are the parent of our subprocess
+        self.assertEqual(error, b'')
         self.assertEqual(int(stdout), os.getpid())
```

Comment:
Note that this test will fail in a virtual environment in which `sys.executable` is the path of a venv launcher. That's the default on Windows instead of using symlinks. The test should run `sys._base_executable` to ensure the child's parent is the current process instead of a launcher.